mlflow/tracing/provider.py

```diff
@@ -215,15 +215,26 @@ def f():
 
     @functools.wraps(f)
     def wrapper(*args, **kwargs):
-        was_trace_enabled = _is_enabled()
-        if was_trace_enabled:
-            disable()
-            try:
+        _is_func_called = False
```

Comment:
```suggestion
        is_func_called = False
```

because it's used. `_xxx_yyy` looks like "it's not used".